todo_app/app.py

```diff
@@ -1,23 +1,35 @@
 import flask
-import todo_app.data.session_items as session_items
 
-from todo_app.flask_config import Config
+from todo_app.config import FlaskConfig, TrelloConfig
+from todo_app.trello import Trello
 
 app = flask.Flask(__name__)
-app.config.from_object(Config)
+app.config.from_object(FlaskConfig)
 
+trello = Trello(TrelloConfig)
 
 @app.route('/')
 def index():
-    return flask.render_template('index.html', tasks = session_items.get_items())
+    return flask.render_template('index.html', lists = trello.get_lists_with_cards())
+
+
+@app.route('/items/<id>/edit', methods=['GET'])
+def edit_item_page(id):
+    return flask.render_template('item_edit.html', item = trello.get_item(id), lists = trello.get_lists())
 
 
 @app.route('/items', methods=['POST'])
-def add_task():
+def add_item():
     title = flask.escape(flask.request.form['title'])
```

Comment:
Worth noting that this is causing unpleasant printing of variables on your webpage e.g. when I added a todo with an apostrophe in. I think you'll find that Jinja & Trello will already escape any content sensibly